pkg/server/server.go

```diff
@@ -300,9 +300,34 @@ func (s *Server) HTTPHandler() http.Handler {
 	handle("/api/console/monitoring-dashboard-config", authHandler(s.handleMonitoringDashboardConfigmaps))
 	handle("/api/console/version", authHandler(s.versionHandler))
 
-	handle("/api/helm/template", authHandlerWithUser(s.handleHelmRenderManifests))
-	handle("/api/helm/release", authHandlerWithUser(s.handleHelmInstall))
-	handle("/api/helm/releases", authHandlerWithUser(s.handleHelmList))
+	// Helm Endpoints
+	handle("/api/helm/template", authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
+		h := handlers.HelmHandlers{
```

Comment:
this here does not make `server.go` more readable than before, quite contrary. I took freedom to try to move the code fully out of this package, hope you are ok with that. Please take a look at PR https://github.com/akashshinde/console/pull/1 adapt if needed.

Reply:
Yeah, I was expecting to see more like:

```go
import "helm/handlers"

handle("/api/helm/template", authHandlerWithUser(helm.handleHelmRenderManifests))
handle("/api/helm/release", authHandlerWithUser(helm.handleHelmInstall))
handle("/api/helm/releases", authHandlerWithUser(helm.handleHelmList))
```
I don't think duplicating this over and over helps:

```go
h := handlers.HelmHandlers{
  // same values each time
}
``` 
I imagine you could set these up as a `func` that returns a `func`:

```go
type helmConfig struct {
  // your 3 values
}
// pass it to a func that will return your funcs with the values needed in the closure 
handle("/api/helm/template", authHandlerWithUser(helm.handleHelmRenderManifests(helmConfig)))
handle("/api/helm/release", authHandlerWithUser(helm.handleHelmInstall(helmConfig)))
handle("/api/helm/releases", authHandlerWithUser(helm.handleHelmList(helmConfig)))
```

Reply:
For context, I think `server.go` acting like a `router` is great.  

```go
handle('/path/to/handle', somepkg.HandleThis)
handle('/other/path/to/handle', somepkg.HandleThat(cfg)) // HandleThat is a func returning a func
```
`/paths` exist here, handler logic within the packages.